build.gradle

```diff
@@ -274,6 +303,8 @@ task buildBeta5(dependsOn: tasks.findAll { t -> t.name.endsWith("-b5")})
 
 task buildRC1(dependsOn: tasks.findAll { t -> t.name.endsWith("-rc1")})
 
+task buildRelease(dependsOn: tasks.findAll { t -> t.name.endsWith("-release")})
```

Comment:
Would prefer naming it `buildLatestRelease(dependsOn: tasks.findAll { t -> t.name.endsWith("-2.0.0")})`. We could move this target to 2.1.0 easily  to always point to the latest release.